### PR TITLE
Migration to `metaschema-xslt`

### PR DESCRIPTION
**Pull request by nikitawootten-nist**
# Committer Notes

Prunes content moved to `metaschema-xslt`.

- [x] Test suite xslt helpers moved to `metaschema-xslt` - note the test suite has not been integrated yet.

### All Submissions:

- [x] Have you followed the guidelines in our [Contributing](https://github.com/usnistgov/metaschema/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/usnistgov/metaschema/pulls) for the same update/change?
- [x] Have you squashed any non-relevant commits and commit messages? \[[instructions](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History)\]
- [x] Do all automated CI/CD checks pass?

### Changes to Core Features:

- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you included examples of how to use your new feature(s)?
- [x] Have you updated all website](https://pages.nist.gov/metaschema) and readme documentation affected by the changes you made? Changes to the website can be made in the website/content directory of your branch.

### CODE REVIEW

**Comment by nikitawootten-nist** [comment]
@david-waltermire-nist Currently building the site yields the error:
```
Error: error building site: POSTCSS: failed to transform "css/hugo-uswds.css" (text/css): env: node: No such file or directory
```

This also happens in `develop` (the only difference being a migration to go modules, they are both pointing at the same commit https://github.com/usnistgov/hugo-uswds/tree/cc57ce2e1efa8602d13160e98090bcacc8e6af93)

**Comment by david-waltermire-nist** [comment]
@nikitawootten-nist I'll look into the website issue.

We may want to keep the support/xml and support/json directories, which provide base schema support. Otherwise, I like what you are removing.

**Comment by nikitawootten-nist** [comment]
@david-waltermire-nist done. @wendellpiez and I were unsure which repo that content should live, but we see an argument for keeping it here.

**Comment by david-waltermire-nist** [comment]
@nikitawootten-nist Would you please rebase to pickup the changes from #398? Thanks.

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,38 +1,25 @@
...
     schedule:
       interval: "daily"
-    target-branch: "develop"
+    target-branch: "main"
 
   - package-ecosystem: "npm"
+    directory: /
     schedule:
       interval: "daily"
     target-branch: "develop"
```
Comment (david-waltermire-nist) [inline .github/dependabot.yml:25]: This is also used by the website, but needs to be at the top-level for Hugo to find it.

```suggestion
    target-branch: "main"
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -1,38 +1,25 @@
...
 
-  - package-ecosystem: "maven"
-    directory: "/build"
+  - package-ecosystem: "gomod"
+    directory: "/website"
     schedule:
       interval: "daily"
-    target-branch: "develop"
+    target-branch: "main"
```
Comment (david-waltermire-nist) [inline .github/dependabot.yml:20]: I don't believe that Go is used outside of the website. Can this be reduced to a single entry?

**Comment by david-waltermire-nist** [comment]
@nikitawootten-nist I removed the website build changes and moved these to PR #422. This should clear this PR for merging.

File: .github/dependabot.yml (hunk 1 above)
Comment (david-waltermire-nist) [reply .github/dependabot.yml:22]: On second thought, I don't believe we should be making any changes to main directly. They should flow through `develop`.

File: .github/dependabot.yml (hunk 2 above)
Comment (david-waltermire-nist) [reply .github/dependabot.yml]: We can address this in #422.

**Comment by david-waltermire-nist** [comment]
@nikitawootten-nist Have the Test suite xslt helpers moved to `metaschema-xslt`? If so, can you check the box above in the PR description?

**Comment by david-waltermire-nist** [comment]
@nikitawootten-nist Also, if you believe this is ready, can you remove the `[draft]` from the title?

**Comment by nikitawootten-nist** [comment]
> @nikitawootten-nist Have the Test suite xslt helpers moved to `metaschema-xslt`? If so, can you check the box above in the PR description?

Yep, although the test suite helpers have not been wired yet. Otherwise I think we're ready to merge!

**Review by david-waltermire-nist — APPROVED** [review]
LGTM. Thanks for doing this!

**Pull request merged**